roles/ngi-methylseq/tasks/main.yml

```diff
@@ -1,11 +1,15 @@
 ---
 
-- name: Fetch NGI-MethySeq from GitHub
+- name: Fetch NGI-MethylSeq from GitHub
   git: repo="{{ ngi_methylseq_repo }}"
        dest="{{ ngi_methylseq_dest }}"
        version="{{ ngi_methylseq_version }}"
        force=yes
 
+- name: Fetch NGI-MethylSeq Singularity Image
```

Comment:
Change to 
```
- name: Create NGI-MethylSeq Singularity folder
  file: name="{{ ngi_resources }}/singularity/ngi-methylseq" state=directory mode=g+s

- name: Pull NGI-MethylSeq Singularity image
  command: "singularity pull {{ ngi_methylseq_docker }}:{{ ngi_methylseq_version }}"
  environment:
    SINGULARITY_CACHEDIR: "/scratch"
  args:
    chdir: "{{ ngi_resources }}/singularity/ngi-methylseq/"
```

Reply:
Great stuff! To confirm, what's the resulting filename? Definitely matches this?

```
{{ ngi_resources }}/singularity/ngi-methylseq/ngi-methylseq-{{ ngi_methylseq_version }}.img
```

Reply:
Yep. `ngi-methylseq-0.3.1.img`